corehq/apps/locations/fixtures.py

```diff
@@ -113,22 +113,23 @@ def __call__(self, restore_state):
         if not should_sync_locations(restore_state.last_sync_log, locations_queryset, restore_state):
             return []
 
-        data_fields = _get_location_data_fields(restore_user.domain)
-        return self.serializer.get_xml_nodes(self.id, restore_user, locations_queryset, data_fields)
+        return self.serializer.get_xml_nodes(restore_user.domain, self.id, restore_user.user_id,
+                                             locations_queryset)
 
 
 class HierarchicalLocationSerializer(object):
 
     def should_sync(self, restore_user, app):
         return should_sync_hierarchical_fixture(restore_user.project, app)
 
-    def get_xml_nodes(self, fixture_id, restore_user, locations_queryset, data_fields):
+    def get_xml_nodes(self, domain, fixture_id, user_id, locations_queryset):
         locations_db = LocationSet(locations_queryset)
 
-        root_node = Element('fixture', {'id': fixture_id, 'user_id': restore_user.user_id})
+        root_node = Element('fixture', {'id': fixture_id, 'user_id': user_id})
```

Comment:
non-actionable aside: I'm actually not sure why we put the user_id in there. It's already in the registration element and doesn't seem like it'd be at all useful.

Reply:
Makes it easier to refer it during iteration on mobile? may be.